test/interchain/README.md

```diff
@@ -3,7 +3,7 @@
 This folder contains tests that use [interchaintest](https://github.com/strangelove-ventures/interchaintest) to assert IBC features work as expected. Candidates for testing include:
 
 1. Interchain Accounts (ICA)
-1. Packet Forward Middleware (PFM)
+2. Packet Forward Middleware (PFM)
```

Comment:
This isn't a typo. It's possible to have numbered lists like this in markdown. https://www.markdownguide.org/basic-syntax/#ordered-lists

```suggestion
1. Packet Forward Middleware (PFM)
```